chaosinstana/api.py

```diff
@@ -8,6 +8,30 @@
 __all__ = ['get_all_events', 'get_event']
 
 
+def get_request(url: str, params: dict, secrets: Secrets) -> str:
```

Comment:
this says it returns a string. But is that the case? Since you return the json response, I think it's likely a `dict` or a `list` response instead?

see https://docs.python.org/3/library/typing.html#module-typing